### PR TITLE
publish bower for buildbot-data

### PR DESCRIPTION
**Pull request by tardyp**
Fixes for the js build scripts, and add publish version

I used this script to push a new tag

https://github.com/buildbot/buildbot-data-js/tree/1.0.15

### CODE REVIEW

**Comment by tardyp** [comment]
@tothandras the publish scripts for buildbot-data, adapted from guanlecoja-ui

File: www/data_module/README.md [hunk 1]
```diff
@@ -25,3 +25,17 @@ $ gulp
 $ npm install
 $ karma start
 ```
+
+## How to test within buildbot/www/base ?
+
+```
+$ cp dist/* ../base/libs/buildbot-data/dist/
```
Comment (tothandras) [inline www/data_module/README.md]: I usually use symlink instead, so I can work both on the base and on the data service

File: www/data_module/guanlecoja/config.coffee [hunk 2]
```diff
@@ -41,9 +59,15 @@ config =
                 version: '~1.1.0'
                 files: 'dist/latest/Dexie.js'
         testdeps:
+            angular:
```
Comment (tothandras) [inline www/data_module/guanlecoja/config.coffee]: Does this pass for you? I've tried to do the same, but failed.

File: www/data_module/guanlecoja/config.coffee [hunk 3]
```diff
@@ -41,9 +59,15 @@ config =
...
+                version: ANGULAR_TAG
+                files: 'angular.js'
             'angular-mocks':
                 version: ANGULAR_TAG
                 files: 'angular-mocks.js'
+    # as angular is a test deps, the tests need to be loaded first!
+    karma:
+        files: [ "tests.js", "scripts.js"]
```
Comment (tothandras) [inline www/data_module/guanlecoja/config.coffee]: I see... :)

**Comment by tothandras** [comment]
@tardyp looks good

File: www/data_module/README.md (hunk 1 above)
Comment (tardyp) [inline www/data_module/README.md]: are you sure this works?

guanlecoja is supposed to erase the dist directory at startup, which will delete your symlink.

Comment (tothandras) [inline www/data_module/README.md]: my workflow looks something like this:
1. run `gulp prod` in base (dependencies are installed)
2. run `gulp prod` in data
3. create symlink
4. run `gulp dev proxy` in base
5. run `gulp dev` in data

**Comment by tardyp** [comment]
shall we merge this?

**Comment by tothandras** [comment]
Looks good!

**Pull request merged**